qiskit_nature/algorithms/ground_state_solvers/adapt_vqe.py

```diff
@@ -42,13 +46,19 @@
 class AdaptVQE(GroundStateEigensolver):
     """A ground state calculation employing the AdaptVQE algorithm."""
 
+    @deprecate_arguments(
+        "0.4.0",
+        {"delta": "gradient"},
+        additional_msg="Instead of Delta=1.0 you have to construct gradient. For example:: Gradient(grad_method='lin_comb', epsilon=1.0)",
```

Comment:
```suggestion
        additional_msg="Instead of `delta=1.0` you have to construct a gradient, like so `gradient=Gradient(grad_method='fin_diff', epsilon=1.0)`.",
```

Reply:
After applying this you should run `make black` to fix the line length.